doc/source/programs/gdal_driver_gti_create.rst

```diff
@@ -0,0 +1,242 @@
+    it already exists.
+
+.. option:: --overwrite-layer
+
+    Whether overwriting the existing output vector layer is allowed.
+
+.. option:: --append
+
+    Whether appending features to the existing output vector layer is allowed
+
+.. option:: -l, --nln, --layer <LAYER>
+
+    Provides a name for the output vector layer.
+
+.. option:: --recursive
+
+    Whether input directories should be explored recursively.
+
+.. option:: --filename-filter <FILENAME-FILTER>
+
+    Pattern that the filenames contained in input directories should follow.
+    '*' is a wildcard character that matches any number of any characters
+    including none. '?' is a wildcard character that matches a single character.
+    Comparisons are done in a case insensitive way.
+    Several filters may be specified.
+
+    For example: ``--filename-filter "*.tif,*.tiff"``
+
+.. option:: --min-pixel-size <val>
+
+    Minimum pixel size in term of geospatial extent per pixel (resolution) that
+    a raster should have to be selected. The pixel size
+    is evaluated after reprojection of its extent to the target SRS defined
+    by :option:`--dst-crs`.
+
+.. option:: --max-pixel-size <val>
+
+    Maximum pixel size in term of geospatial extent per pixel (resolution) that
+    a raster should have to be selected. The pixel size
+    is evaluated after reprojection of its extent to the target SRS defined
+    by :option:`--dst-crs`.
+
+.. option:: -location-name <LOCATION-NAME>
+
+    The output field name to hold the file path/location to the indexed
+    rasters. The default field name is ``location``.
+
+.. option:: --absolute-path
+
+    The absolute path to the raster files is stored in the index file.
+    By default the raster filenames will be put in the file exactly as they
+    are specified on the command line.
+
+.. option:: --dst-crs <DST-CRS>
+
+    Geometries of input files will be transformed to the desired target
+    coordinate reference system.
+    Default creates simple rectangular polygons in the same coordinate reference
+    system as the input rasters.
+
+.. option:: --metadata <KEY>=<VALUE>
+
+    Write an arbitrary layer metadata item, for formats that support layer
+    metadata.
+    This option may be repeated.
+
+
+.. option:: --xml-filename <name>
+
+    Filename of the XML Virtual Tile Index file to generate, that can be used
+    as an input for the GDAL GTI / Virtual Raster Tile Index driver.
+    This can be useful when writing the tile index in a vector format that
+    does not support writing layer metadata items.
+
+.. option:: --resolution <xres,<yres>
+
+    Target resolution in SRS unit per pixel.
+
+    Written in the XML Virtual Tile Index if :option:`--xml-filename`
+    is specified, or as ``RESX`` and ``RESY`` layer metadata items for formats that
+    support layer metadata.
+
+.. option:: --bbox <xmin>,<ymin>,<xmax>,<ymax>
+
+    Target extent in SRS unit.
```

Comment:
Target extent in CRS units. X means easting or longitude, Y northing or latitude.

Reply:
fixed